Source/SwipeToRevealView.swift

```diff
@@ -1,8 +1,17 @@
 import UIKit
 import SnapKit
 
+@objc public protocol SwipeToRevealViewDelegate {
+  @objc optional func swipeToRevealView(view: SwipeToRevealView, didClose animated: Bool)
+  @objc optional func swipeToRevealView(view: SwipeToRevealView, didRevealRight animated: Bool)
+  @objc optional func swipeToRevealView(view: SwipeToRevealView, didPan pan: UIPanGestureRecognizer)
+}
+
+
 /// Swipe-to-reveal view
-public class SwipeToRevealView: UIView {
+open class SwipeToRevealView: UIView {
```

Comment:
Not sure if it's a good idea. This makes the class subclassable. Subclassing is bad, we don't want that.